examples/image_classifier/resnet_18/debugging_backend/test_handler.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+import io
+import logging
+from pathlib import Path
+
+import torch
+
+from ts.torch_handler.image_classifier import ImageClassifier
+from ts.torch_handler.unit_tests.test_utils.mock_context import MockContext
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG)
+
+CURR_FILE_PATH = Path(__file__).parent.absolute()
+REPO_ROOT_DIR = CURR_FILE_PATH.parent.parent.parent.parent
```

Comment:
nit: You can also just use CURR_FILE_PATH.parents[3] here

Reply:
Done